lib/optimizely.rb

```diff
@@ -388,39 +388,37 @@ def get_feature_variable_for_type(feature_flag_key, variable_key, variable_type,
 
       variable_value = nil
```

Comment:
Remove this. No use of assigning nil to it at this point

Reply:
Done!